index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+ <nav class="container">
+  <div class="img">
+  <img src="images/logo.svg" alt="">
+  </div>
+  <div class="navbar">
+    <ul>
+      <li class="item"><a href="#">Home</a></li>
+      <li class="item"><a href="#">New</a></li>
+      <li class="item"><a href="#">Popular</a></li>
+      <li class="item"><a href="#">Trending</a</li>
+      <li class="item"><a href="#">Cartegories</a></li>
+    </ul>
+  </div>
+ </nav>
+ <header class="container">
+  <div class="content">
+    <div class="tem">
+      <img src="images/image-web-3-desktop.jpg" alt="">
+    </div>
+    <div class="sec">
+      <h1>New</h1>
+      <h4>Hydrogen VS Electricity Cars</h4>
+      <p>will Hydrogen-fueled cars catch up to EVs?</p>
+      <br>
+      <hr>
+      <br>
+      <h4>The Downsides of AI artistry</h4>
+      <p>What are possible adverse effects of on-demand Ai image generation?</p>
+      <br>
+      <hr>
+      <br>
+      <h4>Is VC Funding Drying Up?</h4>
+      <p>Private funding by VC films of down 50% YOY. We take a look at the means </p>   
+    </div>
+    <div class="thd">
+      <h1>The Bright Future of Web 3.0?</h1>
+    </div>
+    <div class="fth">
+      <p>We dive into the evolution of the web that claims to
+          put the power of the platforms back into the hands of the people.
+          But is it really fulfilling its promise?
```

Comment:
Kindly make sure that classes are named professionally example fifth_content.
```suggestion
    <div class="fth">
      <p>We dive into the evolution of the web that claims to
          put the power of the platforms back into the hands of the people.
          But is it really fulfilling its promise?
```